scripts/js/lib/api/updateLinks.ts

```diff
@@ -67,36 +69,46 @@ export function normalizeUrl(
 
   const urlParts = url.split("/");
   const initialUrlParts = initial(urlParts);
-  const [path, hash] = last(urlParts)!.split("#") as [
+  const [page, hash] = last(urlParts)!.split("#") as [
     string,
     string | undefined,
   ];
 
+  const normalizedPage = kwargs.kebabCaseAndShorten
+    ? kebabCaseAndShortenPage(page, kwargs.pkgName)
+    : page;
+  const normalizedBase = [...initialUrlParts, normalizedPage].join("/");
```

Comment:
Lmk if you can think of a better name for this. It's the entire normalized URL minus the anchor.